lib/src/widgets/term_of_service_check.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+import '../../flutter_login.dart';
+
+class TermCheck extends StatefulWidget {
+  final TermOfService termOfService;
+
+  const TermCheck({Key? key, required this.termOfService}) : super(key: key);
+
+  @override
+  _TermCheckState createState() => _TermCheckState();
+}
+
+class _TermCheckState extends State<TermCheck> {
+  @override
+  Widget build(BuildContext context) {
+    return CheckboxFormField(
+      onChanged: (value) => widget.termOfService.setStatus(value!),
+      initialValue: widget.termOfService.initialValue,
+      title: widget.termOfService.linkUrl != null
+          ? InkWell(
+              onTap: () {
+                launch(widget.termOfService.linkUrl!);
```

Comment:
Right now it's not clear wether a listtile is an URL or just a regular checkboxlisttile. It should be clear that you can press the text and open a webpage. Maybe its an option to create a trailing webpage icon which can be clicked, or make the text blue like a hyperlink.